### ISSUE TITLE
License

### ISSUE THREAD

**Issue by ajbonner** [DigitalCloud/aramex#2]
Hey there, thanks so much for this code. We have used this as a starting point for our own Aramex API integration and I want to ensure I correctly attribute you, and comply with any license terms. 

There doesn't appear to be any LICENSE.txt attached to the project, but in the composer.json it suggests MIT. Could you add a LICENSE.txt file with date and owner so we can include this with our own version?

Basically you can get the text from here - https://opensource.org/licenses/MIT

Thank you again!
Aaron.
